src/commands/scan/create.ts

```diff
@@ -141,8 +155,7 @@ async function setupCommand(
     cli.showHelp()
     return
   }
-  const { 0: orgSlug = '' } = cli.input
```

Comment:
I need to munch on this a bit longer. I think this is backwards breaking. I realize now that this wasn't written by a JS dev and they didn't realize this is not how you destructure an array in JS ...